### PR TITLE
Updates hitbox expander tool to python3

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

The hitbox expander tool is now python3 instead of python2.
Removed the third party package from the repo, you will now have to rely on your package manager.
Adds scripts to install the dependency.
Fixed an edge case where the tool failed if the metadata value was a number.

### CODE REVIEW

**Comment by psykzz** [comment]
Should really just use a requirements file.

`python3 -m pip freeze > requirements.txt`.
This ensures the correct versions are installed.

Then in your `install.{bat,sh} just `python3 -m pip install requirements.txt``

File: tools/HitboxExpander/hitbox_expander.py [hunk 1]
```diff
@@ -65,8 +65,8 @@ def add(x, y):
       pix[coords] = (0, 0, 0, 1)
 
     PngSave(im, path)
-  #except:
-  #  print("Could not process " + name)
+  except:
+    print("Could not process " + name)
```
Comment (psykzz) [inline tools/HitboxExpander/hitbox_expander.py]:
```suggestion
  except Exception as e:
    print("Could not process " + name)
    print(e)
```

No reason not to post the exception too.

File: tools/HitboxExpander/install.bat [hunk 2]
```diff
@@ -0,0 +1 @@
+python3 -m pip install requirements.txt
```
Comment (psykzz) [inline tools/HitboxExpander/install.bat:1]:
```suggestion
python3 -m pip install -r requirements.txt
```

Sorry if i wasn't clear before

File: tools/HitboxExpander/install.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+python3 -m pip install requirements.txt
```
Comment (psykzz) [inline tools/HitboxExpander/install.sh:1]:
```suggestion
python3 -m pip install -r requirements.txt
```

**Comment by nemvar** [comment]
Oops

**Comment by actioninja** [comment]
All these random tools should probably be cleaned up and stuck in their own repos but this is solid for now

**Pull request merged**